microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -34,6 +34,102 @@ When hosting the WebView2 control, your app has access to the following features
 This page only lists APIs that are in Release SDKs; it doesn't list Experimental APIs, or Stable APIs that are not yet available in Release SDKs.  For a comprehensive list of APIs including Experimental APIs, see [Release Notes for the WebView2 SDK](../release-notes/index.md).
 
 
+<!-- ====================================================================== -->
+## incoming
+
+<!-- ---------- -->
+* Added a new API to provide hit-testing results on the regions that a WebView2 contains.  This API is useful for visually hosted applications that want to handle mouse events on the non-client area of the WebView2 window.
```

Comment:
What section does this API fit it?
Outline: topmost comment in PR main page:
https://github.com/MicrosoftDocs/edge-developer/pull/3096
Rendered: 
https://review.learn.microsoft.com/microsoft-edge/webview2/concepts/overview-features-apis?branch=pr-en-us-3096#incoming

Reply:
idea: find "mouse", "click", in outline

Reply:
Moved hit-testing section to:
```
## Browser features
...
#### Audio
#### Hit-testing of mouse-clicks in regions
#### Swipe gesture navigation
...
```
@victorhuangwq @peiche-jessica